### PR TITLE
[MAINT] Improves onboarding experience

### PR DESCRIPTION
**Pull request by bmjen**
This PR adds the following:
- Improves readability of the logging for evaluations
- Updates README to direct user toward the built-in sample app and docker-compose environment
- Changes the .env templates to point to public api vs internal test api.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,8 @@ Harness Feature Flag PHP SDK
...
 ## Intro
 
-Harness Feature Flags (FF) is a feature management solution that enables users to change the software’s functionality, without deploying new code. FF uses feature flags to hide code or behaviours without having to ship new versions of the software. A feature flag is like a powerful if statement.
-* For more information, see https://harness.io/products/feature-flags/
-* To read more, see https://ngdocs.harness.io/category/vjolt35atg-feature-flags
-* To sign up, https://app.harness.io/auth/#/signup/
+Use this README to get started with our Feature Flags (FF) SDK for PHP. This guide outlines the basics of getting started with the SDK and provides a full code sample for you to try out. 
+This sample doesn’t include configuration options, for in depth steps and configuring the SDK, for example, disabling streaming or using our Relay Proxy, see the PHP SDK Reference.
 
 ![FeatureFlags](https://github.com/harness/ff-php-server-sdk/raw/main/docs/images/ff-gui.png)
```
Comment (bmjen) [inline README.md:17]: The full url is required here, since Packagist scrapes the README from github, so relative paths don't work.

**Review by Clare-Wilkinson — APPROVED** [review]
Looks good from docs perspective 👍

**Review by knagurski — COMMENTED** [review]
Super minor stuff 👍🏻

**Review by knagurski — APPROVED** [review]
DAMN fine work!

**Review by davejohnston — APPROVED** [review]
This looks good.

**Pull request merged**